pages/common/git-imerge.md

```diff
@@ -19,6 +19,10 @@
 
 `git imerge continue --no-edit`
 
-- Wrap up the imerge operation, after all conflicts are resolved:
+- Wrap up imerge operation, after all conflicts are resolved:
 
 `git imerge finish`
+
+- Abort imerge operation, and return to the previous branch:
+
+`git-imerge remove && git checkout {{original_branch}}`
```

Comment:
Oh! I didn't even realize I had used two different terms xD do you prefer "previous" to "original"? We could change either to match the other.